### ISSUE TITLE
Device units are no longer visible on the user instance since version 239

### ISSUE THREAD

**Issue by aroig** [systemd/systemd#9518]
**systemd version the issue has been seen with**
> 239

**Used distribution**
 > Arch Linux

**Expected behaviour you didn't see**
 > device units appear on the `systemctl --user` listing, and other units can declare dependencies on them.

**Unexpected behaviour you saw**
 > device units no longer show up in the user instance.

**Steps to reproduce the problem**
Just do `systemctl --user`, there are no device units listed. ttys, pci devices, etc appeared there in earlier versions of systemd.

**Comment by keszybz** [comment]
Indeed, I can confirm this.

**Comment by chiraag-nataraj** [comment]
I still see device units in Debian (also version 239):
```
$ systemctl --user --no-pager --type device
UNIT                                                              LOAD   ACTIVE     SUB       DESCRIPTION                        
dev-bus-usb-001-024.device                                        loaded active     plugged   dev-bus-usb-001-024.device         
dev-bus-usb-001-025.device                                        loaded active     plugged   dev-bus-usb-001-025.device         
dev-bus-usb-001-026.device                                        loaded active     plugged   dev-bus-usb-001-026.device         
dev-mapper-Crypto\x2dRoot.device                                  loaded activating tentative /dev/mapper/Crypto-Root            
dev-mapper-Crypto_Data\x2dHome.device                             loaded activating tentative /dev/mapper/Crypto_Data-Home       
dev-nvme0n1p1.device                                              loaded activating tentative KXG50ZNV512G NVMe TOSHIBA 512GB ESP
dev-nvme0n1p3.device                                              loaded activating tentative KXG50ZNV512G NVMe TOSHIBA 512GB 3  
sys-devices-pci0000:00-0000:00:14.0-usb1-1\x2d3-1\x2d3:1.0.device loaded active     plugged   Yubikey NEO(-N) OTP+U2F+CCID       
sys-devices-pci0000:00-0000:00:14.0-usb1-1\x2d3-1\x2d3:1.1.device loaded active     plugged   Yubikey NEO(-N) OTP+U2F+CCID       
sys-devices-pci0000:00-0000:00:14.0-usb1-1\x2d3-1\x2d3:1.2.device loaded active     plugged   Yubikey NEO(-N) OTP+U2F+CCID       
sys-devices-pci0000:00-0000:00:14.0-usb1-1\x2d3.device            loaded active     plugged   Yubikey NEO(-N) OTP+U2F+CCID       

LOAD   = Reflects whether the unit definition was properly loaded.
ACTIVE = The high-level unit activation state, i.e. generalization of SUB.
SUB    = The low-level unit activation state, values depend on unit type.

11 loaded units listed. Pass --all to see loaded but inactive units, too.
To show all installed unit files use 'systemctl list-unit-files'.
```

**Comment by aroig** [comment]
I took an other look and today I see only one device, the device for the root partition. It appears in activating state, which is wrong. I am still missing all the other devices.

    systemctl --user --no-pager --type device
    UNIT            LOAD   ACTIVE     SUB       DESCRIPTION                        
    dev-sda2.device loaded activating tentative Samsung_SSD_850_EVO_1TB galois:root

    LOAD   = Reflects whether the unit definition was properly loaded.
    ACTIVE = The high-level unit activation state, i.e. generalization of SUB.
    SUB    = The low-level unit activation state, values depend on unit type.

**Comment by poettering** [comment]
Fix waiting in #10343

**Comment by keszybz** [comment]
@aroig please verify.
